util/verifybackfill.py

```diff
@@ -19,7 +19,7 @@ def _vs(first, second):
 
 
 def verify_backfill(namespace_name):
-    logger.info("Checking namespace %s", namespace_name)
+    logger.debug("Checking namespace %s", namespace_name)
```

Comment:
This file should have been deleted in my cleanup PR.... Please just delete it or revert this change and I'll delete it in a followup